foodsaving/history/api.py

```diff
@@ -1,16 +1,17 @@
 from rest_framework import viewsets
 from rest_framework.filters import DjangoFilterBackend
-from rest_framework.pagination import LimitOffsetPagination
+from rest_framework.pagination import CursorPagination
 from rest_framework.permissions import IsAuthenticated
 
 from foodsaving.history.filters import HistoryFilter
 from foodsaving.history.models import History
 from foodsaving.history.serializers import HistorySerializer
 
 
-class HistoryPagination(LimitOffsetPagination):
-    default_limit = 50
-    max_limit = 1000
+class HistoryPagination(CursorPagination):
```

Comment:
Will this work with the existing frontend? ... getting the first page would be enough I guess

Reply:
Yes, it should work without problems. The frontend uses the provided URL in the `next` field of the pagination header.